purchase_sale_lot_intercompany/tests/test_purchase_sale_lot_intercompany.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2023 Akretion (https://www.akretion.com).
+# @author Kévin Roche <kevin.roche@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.addons.purchase_sale_stock_inter_company.tests.test_inter_company_purchase_sale_stock import (
+    TestPurchaseSaleStockInterCompany,
+)
```

Comment:
Hi @Kev-Roche 

To avoid the conflict between black, isort and flake8 on the line length for python import, I think the least worst solution is :
```suggestion
from odoo.addons.purchase_sale_stock_inter_company.tests.test_inter_company_purchase_sale_stock import (  # noqa: B950
    TestPurchaseSaleStockInterCompany,
)
```
This should make pre-commit test green again...